boards/arduino-leonardo/board.c

```diff
@@ -21,6 +21,7 @@
 #include "board.h"
 #include "cpu.h"
 #include "irq.h"
+#include "clock.h"
```

Comment:
The other platforms don't provide a `clock.h` public header. For consistency, I would prefer AVR to follow suit. However, e.g. within `cpu.h` you can include internal headers to better structure code. It would be good if the internal headers would be "namespaced" if the use common names, e.g. `avr_clock.h` or `cpu_clock.h` shouldn't conflict with other headers.